Updated HCX installation article with more detailed info about HCX on… (#63220)
MicrosoftDocs/azure-docs: articles/azure-vmware/hybrid-cloud-extension-installation.md

---
title: Install VMware HCX
description: Set up the VMware HCX solution for your Azure VMware Solution private cloud
ms.topic: how-to

> ms.date: 07/15/2020

Reviewer: ```suggestion
ms.date: 09/24/2020
```

## Deploy the VMware HCX OVA on-premises

> 1. Sign into the **AVS HCX Manager** on `https://x.x.x.9` port 443 with **cloudadmin** user credentials and the go to **Support**.

Reviewer: ```suggestion
1. Sign into the **Azure VMware Solution HCX Manager** on `https://x.x.x.9` port 443 with **cloudadmin** user credentials and the go to **Support**.
```

Reviewer: We need to spell out AVS since it's not an official acronym (yet).

After installation, perform the following steps.

> 1. Log on to the on-premises HCX manager at `https://HCXManagerIP:9443` and sign in with **admin** username credentials. 

Reviewer: ```suggestion
1. Log on to the on-premises HCX manager at `https://HCXManagerIP:9443` and sign in with the **admin** username credentials. 
```
